EventFilter/CSCRawToDigi/interface/CSCVTMBHeaderFormat.h

```diff
@@ -15,7 +15,7 @@ class CSCVTMBHeaderFormat {
 public:
   virtual ~CSCVTMBHeaderFormat() {}
   void init() {
-    bzero(this, sizeInWords()*2);
+    bzero(static_cast<void *>(this), sizeInWords()*2);
```

Comment:
@Dr15Jones I guess this one is wrong as well.

Reply:
Yes